ipn/ipnserver/server.go

```diff
@@ -413,10 +415,11 @@ func (s *Server) addActiveHTTPRequest(req *http.Request, ci *ipnauth.ConnIdentit
 //
 // At some point, either before or after Run, the Server's SetLocalBackend
 // method must also be called before Server can do anything useful.
-func New(logf logger.Logf, logID logid.PublicID) *Server {
+func New(logf logger.Logf, logID logid.PublicID, netMon *netmon.Monitor) *Server {
```

Comment:
likewise docs on whether nil means something.

you use "netMon" as the var name here but "mon" earlier. be consistent?

Reply:
Went with `netMon` everywhere.